docs/data-ai/data/hot-data-store.md

```diff
@@ -0,0 +1,164 @@
+---
+linkTitle: "Cache recent data"
+title: "Cache recent data"
+weight: 25
+description: "Cache recent data while continuing to write all data to blob storage."
+type: "docs"
+tags: ["hot data store", "aggregation", "materialized views"]
+icon: true
+images: ["/services/icons/data-capture.svg"]
+viamresources: ["sensor", "data_manager"]
+platformarea: ["data", "cli"]
+date: "2024-12-03"
+---
+
+The hot data store caches a rolling window of recent data while continuing to write all data to blob storage.
+Queries to data stored in this cache execute significantly faster than queries to blob storage.
+
+## Configure
+
+To configure the hot data store, add the `recent_data_store` configuration to your component's data capture settings:
+
+```json {class="line-numbers linkable-line-numbers" data-line="17-19"}
+{
+  "components": [
+    {
+      "name": "sensor-1",
+      "api": "rdk:component:sensor",
+      "model": "rdk:builtin:fake",
+      "attributes": {},
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "method": "Readings",
+                "capture_frequency_hz": 0.5,
+                "additional_params": {},
+                "recent_data_store": {
+                  "stored_hours": 24
```

Comment:
So once again - this is just an example. Not the thing we should tell people to add. You need to mark it as such and probably add some indication that the stored_hours are configurable.

I asked for the example to stay in the docs in some format because the preceding description pulls this nicely together to explain usage of the feature. A tab might work - it's what we do elsewhere.

Reply:
Thanks for the explanation. I didn't understand that you were referring to the sentence and the example snippet before. Sorry for misunderstanding.

Reworded this to explicitly state that the configuration is an example, lifted the sentence about `stored_hours` configuration above the snippet, and added an introductory sentence to help users connect the 24 in the example to 24 stored hours of data.